R/pool-methods.R

```diff
@@ -83,17 +83,6 @@ setMethod("poolClose", "Pool", function(pool) {
   pool$close()
 })
 
-#' Show method
```

Comment:
This didn't appear to do anything, probably because all pool objects were R6/S3 not S4.